tests/circleci_pipelines/image_referencer/test_runner.py

```diff
@@ -34,10 +34,10 @@ def test_github_action_workflow(mocker: MockerFixture, image_cached_result, file
     sca_image_report = next(report for report in reports if report.check_type == CheckType.SCA_IMAGE)
 
     assert len(circleci_report.resources) == 0
-    assert len(circleci_report.passed_checks) == 23
+    assert len(circleci_report.passed_checks) == 20
     assert len(circleci_report.failed_checks) == 13
     assert len(circleci_report.skipped_checks) == 0
     assert len(circleci_report.parsing_errors) == 0
 
-    assert len(sca_image_report.extra_resources) == 9
-    assert len(sca_image_report.image_cached_results) == 9
+    assert len(sca_image_report.extra_resources) == 10
+    assert len(sca_image_report.image_cached_results) == 10
```

Comment:
can you also test here for sca_image_report.image_cached_results`

Reply:
added :)